src/JSInterop/Microsoft.JSInterop/src/Infrastructure/DotNetDispatcher.cs

```diff
@@ -292,6 +295,32 @@ public static void EndInvokeJS(JSRuntime jsRuntime, string arguments)
             }
         }
 
+        /// <summary>
+        /// Accepts the byte array data being transferred from JS to DotNet.
+        /// </summary>
+        /// <param name="jsRuntime">The <see cref="JSRuntime"/>.</param>
+        /// <param name="id">Identifier for the byte array being transfered.</param>
+        /// <param name="data">Byte array to be transfered from JS.</param>
+        public static void SupplyByteArray(JSRuntime jsRuntime, long id, byte[] data)
```

Comment:
I think this method must go in the JSRuntime implementation directly instead of here. It's suffering from `feature envy` as described [here](https://moderatemisbehaviour.github.io/clean-code-smells-and-heuristics/general/g14-feature-envy.html)